action.yml

```diff
@@ -92,32 +95,30 @@ runs:
         else
           echo "Skipping building cdk-validium-node as release tag provided: ${{ inputs.zkevm_node }}"
         fi
-    - name: Install Foundry
-      shell: bash
-      run: |
-        curl -LO https://github.com/foundry-rs/foundry/releases/download/nightly-de33b6af53005037b463318d2628b5cfcaf39916/foundry_nightly_linux_amd64.tar.gz
-        tar -xzvf foundry_nightly_linux_amd64.tar.gz
-    - name: Clone internal kurtosis-cdk repo
-      shell: bash
-      run: |
-        git clone https://github.com/0xPolygon/kurtosis-cdk.git
-        cd kurtosis-cdk 
-        git checkout "${{ inputs.kurtosis_cdk }}"
-    - name: Install kurtosis
+
+    # Install tools
+    - name: Install Kurtosis CDK tools
+      uses: ./.github/actions/setup-kurtosis-cdk
```

Comment:
we've tested relative references like this work for shared git actions?

Reply:
No I haven't tested, that's why I mentioned you in the PR